Kiosk/App/AppDelegate+GlobalActions.swift

```diff
@@ -95,13 +95,13 @@ extension AppDelegate {
 
     func showPrivacyPolicyCommand() -> CocoaAction {
         return CocoaAction { _ in
-            self.hideAllTheThings().then(self.showWebController("https://artsy.net/privacy"))
+            self.hideAllTheThings().then(self.showWebController(address: "https://artsy.net/privacy"))
         }
     }
 
     func showConditionsOfSaleCommand() -> CocoaAction {
         return CocoaAction { _ in
-            self.hideAllTheThings().then(self.showWebController("https://artsy.net/conditions-of-sale"))
+            self.hideAllTheThings().then(self.showWebController(address: "https://artsy.net/conditions-of-sale"))
```

Comment:
So the `hideAllTheThings()` observables is dismissing the fulfillment nav controller, which ultimately causes the button who owns this action to be deallocated, and the Action gets deallocated, and the subscription to this observable chain gets disposed of before the web controller has been presented. Yikes.